### DISCUSSION TITLE
comment/uncomment feature in vscode DSL plugin

### DISCUSSION THREAD

**Discussion by goto40** [eclipse-langium/langium#1140] [Q&A] [answered]
This is somehow only loosely related to langium - but I wonder how I could add/enable the "Toogle Line Comment" feature for my langium vscode plugin of my langium DSL. Probably this just a standard vscode plugin configuration. 

~~Is there already an example? It does not seem to be in the plugin for `*.langium` files (at least the version I installed).
If not I will find out myself...~~

The langium DSL/plugin has the feature. I will look it up there.

**Comment by goto40** [comment, marked as answer]
The langium DSL/plugin has the feature. I will look it up there.

**Reply by msujew** [reply]
FYI, the behavior can the achieved through [this language configuration](https://github.com/langium/langium/blob/13f482bf33169092f17cf92e327b799c21f5b4dc/packages/langium-vscode/data/langium.configuration.json#L2-L5).
